plaso/parsers/sqlite_plugins/tango_android.py

```diff
@@ -0,0 +1,332 @@
+# -*- coding:utf-8 -*-
+"""Parser for Tango on Android databases."""
+
+from __future__ import unicode_literals
+
+import codecs
+
+from dfdatetime import java_time as dfdatetime_java_time
+from dfdatetime import semantic_time as dfdatetime_semantic_time
+
+from plaso.containers import events
+from plaso.containers import time_events
+from plaso.lib import definitions
+from plaso.parsers import sqlite
+from plaso.parsers.sqlite_plugins import interface
+
+
+class TangoAndroidMessageEventData(events.EventData):
+  """Tango on Android message event data.
+
+  Attributes:
+    message_id (int): message id.
+    payload (str): binary blob with message metadata.
```

Comment:
a blob would be of type bytes, otherwise it would be encoded.

However I do not recommend storing the blob in the event.

Reply:
I will remove payload and add a TODO to build and extract the payload structure, ParseMessageRow() won't make much sense without this field.